### PR TITLE
Added support aten::avg_poolNd and aten::max_poolNd with no batch

### PR DESCRIPTION
**Pull request by krish1209**
Two files updated and removed the new files I created.

#20927

### CODE REVIEW

File: src/frontends/pytorch/src/op/Avgpool_check.cpp [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }
+    RoundingType rounding_type;
+    if (context.input_is_none(5)) {
+        rounding_type = RoundingType::FLOOR; // Default rounding type
+    } else {
+        rounding_type = context.const_input<bool>(5) ? RoundingType::CEIL : RoundingType::FLOOR; // Rounding type based on input
+    }
+
+    // Call average pooling operation with extracted parameters
+    auto res = context.mark_node(std::make_shared<v8::AvgPool>(input, // Changed to AvgPool
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/Avgpool_check.cpp:62]: I don't see how you reshape `input` to support `CHW` and `NCHW` cases.

**Review by mvafin — CHANGES_REQUESTED** [review]
Please do not create new files for these operations, rather update old files

**Comment by krish1209** [comment]
Okay I'll update the old files and update the PR. Sorry for the inconvenience.

**Comment by mvafin** [comment]
please also add tests for this functionality and verify that your code works

**Comment by krish1209** [comment]
I am currently facing an "IncludePath" error even after adding the location of the header files so it is taking a little time to debug the same. onnx_pb.h is a file my system is unable to find. I'll get back as soon as possible after solving the error.

Also is it possible for you to guide me a little bit on how i can make a custom test case in the code base for these operations? It would be really helpful.

**Comment by krish1209** [comment]
Hello Sir, I apologize for the absence. I got my head into a Research Paper based project for publishing, so couldn't find the time.
Is it possible for you to mention what all needs to be done in the markdown comment in order to merge this pull request in the main branch. Thank you. I will get at it as soon as possible :)

**Comment by krish1209** [comment]
Can anyone provide an update so that I could be clarified the work I have to do. Thank you :)

**Comment by mvafin** [comment]
> Hello Sir, I apologize for the absence. I got my head into a Research Paper based project for publishing, so couldn't find the time. Is it possible for you to mention what all needs to be done in the markdown comment in order to merge this pull request in the main branch. Thank you. I will get at it as soon as possible :)

- Write tests for pooling with no batch
- Run your code and identify if all cases work
- Fix issues and make sure that tests pass

**Comment by krish1209** [comment]
Okay. Thank you for replying :). I will get on it as soon as possible.

**Pull request closed without merging**